### ISSUE TITLE
`prevent-abbreviation` suggests replacing `someRetVal` with `someReturnValueValue`

### ISSUE THREAD

**Issue by Xunnamius** [sindresorhus/eslint-plugin-unicorn#1937]
Rule `unicorn/prevent-abbreviation` suggests replacing `someRetVal` with `someReturnValueValue`.

Rule: `unicorn/prevent-abbreviation`

```js
const expectedRetVal = 'that should be ok';

// Expected:
const expectedReturnValue = 'that should be ok';

// Actual:
const expectedReturnValueValue = 'that should be ok';
```
